lib/rouge/lexers/objective_cpp.rb

```diff
@@ -3,8 +3,8 @@
 
 module Rouge
   module Lexers
-    load_lexer 'cpp.rb'
-    load_lexer 'objective_c/common.rb'
+    preload_lexer 'cpp'
+    require_relative 'objective_c/common'
```

Comment:
Why `require_relative`? This doesn't look like its compatible for use after calling `Rouge.reload!`

Reply:
That's true - I'll have to rearchitect slightly for that.